cloud/linode/instances.go

```diff
@@ -13,12 +14,46 @@ import (
 	cloudprovider "k8s.io/cloud-provider"
 )
 
+type nodeCache struct {
+	sync.RWMutex
+	nodes      map[int]*linodego.Instance
+	lastUpdate time.Time
+	ttl        time.Duration
+}
+
+func (nc *nodeCache) refresh(ctx context.Context, client Client) error {
```

Comment:
would it make sense to name this `nodeCacheRefresh` or `instanceRefresh`?

Reply:
Note that it's a method on `nodeCache`, so the (only) call site is literally `instances.nodeCache.refresh`, so including `nodeCache` or `instance` in the name would only duplicate prior mentions of these.

Though come to think of it, we do shorten `instances` to `i` when calling it, so we could make it `refreshInstances` and the call site would then be `i.nodeCache.refreshInstances(...)`.